src/pages/LogOutPreviousUserPage.tsx

```diff
@@ -14,24 +13,18 @@ import ONYXKEYS from '@src/ONYXKEYS';
 import ROUTES from '@src/ROUTES';
 import type {Route} from '@src/ROUTES';
 import type SCREENS from '@src/SCREENS';
-import type {Session} from '@src/types/onyx';
 
-type LogOutPreviousUserPageOnyxProps = {
-    /** The data about the current session which will be set once the user is authenticated and we return to this component as an AuthScreen */
-    session: OnyxEntry<Session>;
-
-    /** Is the account loading? */
-    isAccountLoading: boolean;
-};
-
-type LogOutPreviousUserPageProps = LogOutPreviousUserPageOnyxProps & StackScreenProps<AuthScreensParamList, typeof SCREENS.TRANSITION_BETWEEN_APPS>;
+type LogOutPreviousUserPageProps = StackScreenProps<AuthScreensParamList, typeof SCREENS.TRANSITION_BETWEEN_APPS>;
 
 // This page is responsible for handling transitions from OldDot. Specifically, it logs the current user
 // out if the transition is for another user.
 //
 // This component should not do any other navigation as that handled in App.setUpPoliciesAndNavigate
-function LogOutPreviousUserPage({session, route, isAccountLoading}: LogOutPreviousUserPageProps) {
+function LogOutPreviousUserPage({route}: LogOutPreviousUserPageProps) {
     const {initialURL} = useContext(InitialURLContext);
+    const [session] = useOnyx(ONYXKEYS.SESSION);
+    const [account = {}] = useOnyx(ONYXKEYS.ACCOUNT);
+    const isAccountLoading = account?.isLoading;
```

Comment:
Yeah, agree we should make it as before. Wdyt about this suggestion?

```suggestion
    const [account] = useOnyx(ONYXKEYS.ACCOUNT);
    const isAccountLoading = account?.isLoading ?? false;
```

Reply:
Good point